apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -1,5 +1,6 @@
 import type { Meta, StoryObj as Story } from '@storybook/web-components';
-import { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
+import type { ButtonSize, ButtonType, ButtonVairant } from '@justeattakeaway/pie-button';
+import { buttonSizes, buttonTypes, buttonVariants } from '@justeattakeaway/pie-button';
```

Comment:
you can import all of these in one line

Reply:
The internet has provided me with this as an option 
```
import {
    type ButtonSize,
    type ButtonType,
    type ButtonVariant,
    buttonSizes,
    buttonTypes,
    buttonVariants,
} from './defs';
```
If you want it we can do it. I personally think it's messier this way

Reply:
I think it's cleaner personally, however, what happens when you omit `type` and just import it all?

Reply:
It errors cos it doesn't know what it is 
![image](https://github.com/justeattakeaway/pie/assets/24740015/4b12ded4-e237-43fb-b3b8-23f0e16fcff9)

Reply:
Then I'd opt for the joint import you pasted above personally.

Reply:
I'm happy with this. There's no `import interface {...` thing needed for an interface right? So it would just be
```
import { ButtonProps, buttonSizes, buttonTypes, buttonVariants } from './defs';
```

Reply:
yup this looks much better to me 👍

Reply:
🥳 awesome thanks @raoufswe